packages/cli-kit/src/private/node/tree-kill.ts

```diff
@@ -1,7 +0,0 @@
-import {printEventsJson} from './demo-recorder.js'
-import originalTreeKill from 'tree-kill'
-
-export function treeKill(signal: string) {
-  printEventsJson()
-  originalTreeKill(process.pid, signal)
-}
```

Comment:
why did we remove this? did we move it to public?

Reply:
Yes, I believe we did move it to public.